permissions/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "manifest_version": 2,
+  "name": "Permissions Example",
+  "description": "Permissions Example",
+  "version": "1.0",
+
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "icons": {
+    "48": "icon.svg"
+  },
+  "browser_action": {
+    "default_title": "Permissions Example",
+    "default_icon": "icon.svg"
+  },
+
+  "permissions": [
+    "tabs"
+  ],
+  "optional_permissions": [
+    "history"
+  ]
+}
```

Comment:
Since this API is new in Firefox 55, we should probably include an "applications" key here with strict_min_version